celobox/passwd.py

```diff
@@ -31,7 +34,9 @@ def new_session(self):
 
     def load_data(self, domain):
         try:
-            return json.load(open('manifests/%s.json' % domain))
+            return json.load(open(
+                resource_filename(__name__, os.path.join('manifests', '{}.json'.format(domain)))
```

Comment:
There are two runners now: `passwd.py` (this one), and `phantompasswd.py`.
Only the latter works with the newer version of the manifests (in the `phantom` branch). There's no YAML support in `master`.
In short: `phantompasswd.py` will replace `passwd.py`, so this is fine.